src/Model/Contact.php

```diff
@@ -2311,16 +2311,15 @@ public static function getProtocol($url, $network)
 	 *
 	 * Takes a $uid and a url/handle and adds a new contact
 	 *
-	 * @param array  $user        The user the contact should be created for
+	 * @param int    $uid         The user id the contact should be created for
 	 * @param string $url         The profile URL of the contact
-	 * @param bool   $interactive
 	 * @param string $network
 	 * @return array
 	 * @throws HTTPException\InternalServerErrorException
 	 * @throws HTTPException\NotFoundException
 	 * @throws \ImagickException
 	 */
-	public static function createFromProbe(array $user, $url, $interactive = false, $network = '')
+	public static function createFromProbe(int $uid, $url, $network = '')
```

Comment:
Please rename to include the user in it, like
```suggestion
	public static function createFromProbeForUser(int $uid, $url, $network = '')
```

Reply:
Done